application/controllers/UserManagementController.php

```diff
@@ -506,8 +506,10 @@ public function actionUserPermissions()
                 }
             }
             $aBasePermissions = $aFilteredPermissions;
+        } elseif(!Permission::model()->hasGlobalPermission('superadmin', 'create')) {
+            /* superadmin permission always need create */
```

Comment:
Why using multiline comments?
`//` isn't a better fit?

Reply:
It's still an one line comment

Reply:
I think i make a simple version for master : just unset the superadmin in arrau of Permission. I do same here.

I really think it more clear